genny/auth/templates/models/user.go.plush

```diff
@@ -31,15 +35,48 @@ type User struct {
 // Create wraps up the pattern of encrypting the password and
 // running validations. Useful when writing tests.
```

Comment:
Should include a hint to making emails lowercase as well. See comment on `validatePassword` as well